test/oauth-test.js

```diff
@@ -1,73 +1,190 @@
 const oauth = require('../oauth/index');
-const oauthv2 = require('../oauthv2/index');
 const assert = require('assert');
+const denv = require('dotenv');
+denv.config();
 
-const logger = {}
-const stats = {}
 
-describe('oauth plugins', function() {
+var oauthConfiigDefaults = {
+  "authorization-header" : "authorization",
+  "api-key-header" : 'x-api-key',
+  "keep-authorization-header" : false,
+  "cacheKey" : false,
+  "gracePeriod" : 0,
+  "allowOAuthOnly" : false,
+  "allowAPIKeyOnly" : false,
+  "productOnly" : false,
+  "tokenCache" : false,
+  "tokenCacheSize" : 100,
+  "allowNoAuthorization" : false,
+  "jwk_keys" : undefined,
+  "request" : undefined
+}
 
-// should be identical for these tests
```

Comment:
I just added all these tests for ejectToken and it looks like they're being removed... we need to revisit what's going on here.

Reply:
So, I should have seen your tests and done a merge both. I guess that's why I found the configs commented out. It seems we were both working on the same file. So, I will be more wary in the future. Perhaps there is a simple way to alert each other about this sort of case - for instance a rapid issue page or something. But, it is likely not to happen too much and tools might be more trouble than they are worth. I do know of a little chat page that allows you to turn chat lines into issues, but it is not the most popular software.

Reply:
It's not uncommon to work on the same file. In general, git just handles it. But sometimes git can't guess how to handle it... and that's what a conflict tells you. I don't think we need a new tool... just, as you said, be more wary.